include/RE/B/BSTEvent.h

```diff
@@ -102,6 +103,4187 @@ namespace RE
 	public:
 		// members
 		BSTEventSource<Event>* eventSource;  // 00
+		std::uint64_t          unk08;        // 08
 	};
-	static_assert(sizeof(BSTEventSourceLazyInit<void*>) == 0x8);
+	static_assert(sizeof(BSTEventSourceLazyInit<void*>) == 0x10);
+
+
+
+
+	namespace Events
+	{
+		struct TESLoadGameEvent
+		{
+		public:
+			[[nodiscard]] static RE::BSTEventSource<TESLoadGameEvent>* GetEventSource()
+			{
+				REL::Relocation<uintptr_t> funcPtr{ REL::ID(1868757) };
+				uintptr_t                  addr = funcPtr.address();
+				const auto                 func = reinterpret_cast<RE::BSTEventSource<TESLoadGameEvent>* (*)()>(addr);
+				return func();
```

Comment:
i'm not quite sure what all of these are doing. if they're just treating the address a a function to call then this needs to be changed to be in line with all other uses